codecs/text/src/test/java/com/datastax/oss/dsbulk/codecs/text/string/StringToStringCodecTest.java

```diff
@@ -19,16 +19,27 @@
 
 import com.datastax.oss.driver.api.core.type.codec.TypeCodecs;
 import com.datastax.oss.driver.shaded.guava.common.collect.Lists;
-import java.util.List;
+import java.util.Collections;
 import org.junit.jupiter.api.Test;
 
 class StringToStringCodecTest {
 
-  private List<String> nullStrings = Lists.newArrayList("NULL");
-
   @Test
   void should_convert_from_valid_external() {
-    StringToStringCodec codec = new StringToStringCodec(TypeCodecs.TEXT, nullStrings);
+    StringToStringCodec codec = new StringToStringCodec(TypeCodecs.TEXT, Collections.emptyList());
+    assertThat(codec)
+        .convertsFromExternal("foo")
+        .toInternal("foo")
+        .convertsFromExternal("")
```

Comment:
Since this is a textual CQL type, empty strings, and nulls are all left untouched.